doc/requirements.rst

```diff
@@ -156,20 +157,15 @@ Test coverage
     :label_set: not covered, covered
     :colors: orange c b
     :stats:
+    :matrix:
 
-.. item-piechart:: Test coverage chart targeting UTEST and ITEST
+.. item-piechart:: Test coverage chart with test results, based on the :targettype: option
```

Comment:
To keep aligned with `:result: attrubute` below

```suggestion
.. item-piechart:: Test coverage chart with test results, based on the :targettype: attribute
```

Reply:
No. The `:result:` attribute is the `:<<attribute>>:` option. "targettype" is not a traceable attribute, it's an option of this directive.